SpatialVideoGist/SpatialVideoGist/Processing/SpatialVideoConverter.swift

```diff
@@ -144,21 +165,30 @@ import VideoToolbox
             kCVPixelBufferWidthKey as String: naturalSize.width,
             kCVPixelBufferHeightKey as String: naturalSize.height
         ]
-        readerOutput = AVAssetReaderTrackOutput(
+        readerVideoOutput = AVAssetReaderTrackOutput(
             track: videoTrack,
             outputSettings: readerOutputSettings)
+        if let audioTrack {
+            readerAudioOutput = AVAssetReaderTrackOutput(track: audioTrack, outputSettings: nil)
```

Comment:
My audio output settings were the following:
```        // Audio iwriter input settings
        var channelOutputLayout = AudioChannelLayout()
        memset(&channelOutputLayout, 0, MemoryLayout<AudioChannelLayout>.size);
        channelOutputLayout.mChannelLayoutTag = kAudioChannelLayoutTag_Stereo;

        let audioOutputSettings = [
            AVFormatIDKey as String: NSNumber(value: kAudioFormatLinearPCM),
            AVSampleRateKey as String: NSNumber(value: 44100.0),
            AVNumberOfChannelsKey as String: NSNumber(value: 2),
            AVChannelLayoutKey as String: NSData(bytes: &channelOutputLayout, length: MemoryLayout<AudioChannelLayout>.size),
            AVLinearPCMBitDepthKey as String: NSNumber(value: 16),
            AVLinearPCMIsNonInterleaved as String: NSNumber(booleanLiteral: false),
            AVLinearPCMIsFloatKey as String: NSNumber(booleanLiteral: false),
            AVLinearPCMIsBigEndianKey as String: NSNumber(booleanLiteral: false),
        ]
        
        audioReaderOutput = AVAssetReaderTrackOutput(
            track: audioTrack,
            outputSettings: audioOutputSettings)```
I tried `AVFormatIDKey: kAudioFormatMPEG4AAC` before but it yelled at me at run time saying that the audio format could not be a compressed type. So the above settings worked, but I still need to confirm if its actually stereo (it does specify 2 channels, but i haven't confirmed the channels didn't cross somehow)

Reply:
> My audio output settings were the following:
> 
> ```
>         // Audio iwriter input settings
>         var channelOutputLayout = AudioChannelLayout()
>         memset(&channelOutputLayout, 0, MemoryLayout<AudioChannelLayout>.size);
>         channelOutputLayout.mChannelLayoutTag = kAudioChannelLayoutTag_Stereo;
> 
>         let audioOutputSettings = [
>             AVFormatIDKey as String: NSNumber(value: kAudioFormatLinearPCM),
>             AVSampleRateKey as String: NSNumber(value: 44100.0),
>             AVNumberOfChannelsKey as String: NSNumber(value: 2),
>             AVChannelLayoutKey as String: NSData(bytes: &channelOutputLayout, length: MemoryLayout<AudioChannelLayout>.size),
>             AVLinearPCMBitDepthKey as String: NSNumber(value: 16),
>             AVLinearPCMIsNonInterleaved as String: NSNumber(booleanLiteral: false),
>             AVLinearPCMIsFloatKey as String: NSNumber(booleanLiteral: false),
>             AVLinearPCMIsBigEndianKey as String: NSNumber(booleanLiteral: false),
>         ]
>         
>         audioReaderOutput = AVAssetReaderTrackOutput(
>             track: audioTrack,
>             outputSettings: audioOutputSettings)
> ```
> 
> I tried `AVFormatIDKey: kAudioFormatMPEG4AAC` before but it yelled at me at run time saying that the audio format could not be a compressed type. So the above settings worked, but I still need to confirm if its actually stereo (it does specify 2 channels, but i haven't confirmed the channels didn't cross somehow)

Thank you for all of your feedback and investigation on this, @altonelli!  I did some further testing and found that if I don't set *any* audio output settings at all, the audio settings are mirrored from the source file to the destination file (as a test, I used a source file that had multiple audio tracks, and a unique audio mapping configuration, converted to a Spatial Video, and then looked at the `CMFormatDescription` of the output file, validating that all of the audio settings were identical).

Based on this, I am going to merge this PR as is, and in a follow-up, offer the ability to offer more customizations to the output audio and video settings.  I figure that since AVAssetReader appears to handle outputting the audio settings and mapping them to the destination file automatically, that may as well be the default behavior, so there are no hard-coded values (like `kAudioFormatMPEG4AAC` or `kAudioFormatLinearPCM`).

Let me know if you disagree and I can re-open this, or I'll address re-implementing this logic in the follow-up PR that allows for more advanced customizations of video and audio profiles.